### ISSUE TITLE
[FEATURE REQUEST] - PF2e - Autorec by Slug instead of Name

### ISSUE THREAD

**Issue by AlexYukikaze** [otigon/automated-jb2a-animations#651]
Auto-recognition json configs don't work when using babele localization other than english.

The solution to this problem is to use Slugs instead of Names. Slug in PF2e is stable and it will remain in place even if the item was renamed.

**Comment by otigon** [comment]
I've got no idea what you mean by Slug. Is this something in the data structure?

**Comment by AlexYukikaze** [comment]
Yes it is. For example same spell on english and russian have same slug
![image](https://github.com/otigon/automated-jb2a-animations/assets/2861110/f5556feb-4cd9-4695-89d8-74f6df62fb96)
![image](https://github.com/otigon/automated-jb2a-animations/assets/2861110/6b61e522-a11e-40b0-b621-922e62044bfe)

**Comment by Mig287** [comment]
I digged into this to make me a workaround (Issue #661)
And im pretty sure that its unnecesarry. 
I think it should be possible to get all translations from the translation files. But I have no idea how babele and AA are connected. Thats way above my coding knowledge. 
But nonetheless, usind the slug would be an alternative to make it working in every language, but I believe its an pf2e only thing.
